tests/testServer/testServer.cpp

```diff
@@ -1,72 +1,181 @@
 // Copyright 2019 IBM Corporation
-// 
+//
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
-// 
+//
 //     http://www.apache.org/licenses/LICENSE-2.0
-// 
+//
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
 #include <iostream>
+#include <fstream>
 
 #include <grpcpp/grpcpp.h>
 #include "ServiceScalarTypeRpcs.hpp"
 #include "ServiceStreamingRpcs.hpp"
 #include "ServiceComplexTypeRpcs.hpp"
 #include "ServiceNestedTypeRpcs.hpp"
 #include "ServiceStatusHandling.hpp"
+#include <grpcpp/security/credentials.h>
+/// Function for reading and returning credentials (Key, Cert) for secure server
+std::string readFromFile(const char f_path[])
+{
+    std::ifstream credFile(f_path);
+    //credFile.open(f_path);
+    //d::string fileContent;
+    if (credFile)
+    {
+        char nextChar;
+
+        std::string str{std::istreambuf_iterator<char>(credFile),
+                        std::istreambuf_iterator<char>()};
+
+        if (credFile)
+            std::cout << "Server: File content of " << f_path << ": " << str << std::endl;
 
+        return str;
+    }
+    else
+    {
+        std::cout << "No cert/key found at " << f_path << std::endl;
+        std::cout << "Stopping server" << std::endl;
+        exit(EXIT_FAILURE);
+    }
+}
 
 int main(int argc, char **argv)
 {
 
-    if(argc >= 2 and (std::string(argv[1]) == "-h" or std::string(argv[1]) == "--help"))
+    if (argc >= 2 and (std::string(argv[1]) == "-h" or std::string(argv[1]) == "--help"))
     {
-        std::cout << "A simple gRPC test server implementing RPCs using most of the proto3 language features." << std::endl << std::endl;
+        std::cout << "A simple gRPC test server implementing RPCs using most of the proto3 language features." << std::endl
+                  << std::endl;
         std::cout << "SYNOPSIS:" << std::endl;
```

Comment:
documentation outdates. it seems, that we only support passing the PORT argument for insecure connections. please update the help output, or make something like "BasePort" which is used for deriving the three ports i.e. Base=50000 then insecure = Base + 1, secure = Base + 443 etc.